source/_integrations/xiaomi_miio.markdown

```diff
@@ -73,6 +73,14 @@ you used in the Xiaomi Home App (where you initially setup the device). There ar
 [this page](https://www.openhab.org/addons/bindings/miio/#country-servers) for
 the server to use for each country.
 
+## Troubleshooting
+
+[Detailed troubleshooting steps are here](https://python-miio.readthedocs.io/en/latest/troubleshooting.html#troubleshooting)
```

Comment:
We should not point people to libraries in our documentation.

Reply:
I removed that statement.
I hope it is alright to have the `more info` links at the end of each problem in the list?